### ISSUE TITLE
[COGA] 2.6.1 Understanding How to Interact with the Interface

### ISSUE THREAD

**Issue by RealJoshue108** [w3c/apa#278]
[Added on behalf of COGA TF]

* New note: COGA TF recommends emphasizing that the users who need the most support understanding how to interact with the interface may be the least likely to be able to find or to be comfortable changing the accessibility settings. 
* New User Need about working memory: Voice menus place a heavy burden on users’ working memory, which many people struggle with for a broad range of reasons, including situational. 
   * New REQ #1: Enable the user to ask questions out of the flow to try to help them through this process, or give them a way to access reminders.
   * New REQ #2: Rather than try and force a menu structure onto broad-range assistants (which can also be cognitively overwhelming), enable users to ask for what they need in a natural way, and not have to memorize commands. 
   * New REQ #3: Provide "just in time" help, where the system suggests the right option at the right moment—when it's useful for the context the user is in.
   * New REQ #4: Ensure sure the system can answer "how to" questions, so people can ask for things like "How do I change my font settings" or "make the font bigger" rather than having to navigate challenging menu structures etc. or be forced to switch to another device (example: go from a smart speaker to a phone) to complete a task. 
   * New REQ #5: Wherever possible, ensure that the user flow enables the user to complete a task on the same device on which they started it.

**Comment by RealJoshue108** [comment]
ack several new suggested requirements.

**Comment by jchassery12** [comment]
Yes I absolutely will but what make it’s hard is I only have an IPhone 11. Incan not afford a laptop or desktop. So please it would be easier if someone talked me through or sent me a link relevant to an iPhone.

**Comment by RealJoshue108** [comment]
New user need and related requirements request

**Comment by RealJoshue108** [comment]
@jasonjgw Needs to be better defined - @JaninaSajka Issue with the idea of 'out of the flow'

**Comment by RealJoshue108** [comment]
Discussed in Research Questions Task Force 26th Jan 2022 https://www.w3.org/2022/01/26-rqtf-minutes.html
